src/sentry/dynamic_sampling/tasks/boost_low_volume_projects.py

```diff
@@ -309,10 +309,11 @@ def adjust_sample_rates_of_projects(
             )
 
             if rebalanced_project.id in PROJECTS_WITH_METRICS:
-                metrics.timing(
+                metrics.gauge(
                     "dynamic_sampling.project_sample_rate",
                     rebalanced_project.new_sample_rate,
                     tags={"project_id": rebalanced_project.id},
+                    unit="percent",
```

Comment:
@iambriccardo Is this on a 0-1 scale or 0-100?

Reply:
This is in `[0-1]`.

Reply:
So yeah, I should cast to `[0-100]` since I see that frontend is assuming to get `[0-100]`.

Reply:
sounds good
alternatively, `ratio` unit takes a value between 0-1 and formats to percentages on frontend but I think `percent` works better here

Reply:
Yeah, it's semantically closer to what the metric is about.